public/doc/openapi-3-v1.0.0.json

```diff
@@ -1937,7 +1952,22 @@
             "description": "Portfolio Item Icon",
             "content": {
               "image/svg+xml": {
-                "example": "<svg rel=\"stylesheet\">thing</svg>"
+                "schema": {
```

Comment:
@lindgrenj6 Have you been able to test this change with a ruby generated client and fetch an icon?

Reply:
Yes, when using @syncrou's [catalog-api-client-tests](https://github.com/syncrou/catalog-api-client-tests) repo I am able to successfully download the binary data: 
![image](https://user-images.githubusercontent.com/5856504/72288345-7ba3dd00-360e-11ea-8c83-cec238fd7fd6.png)